examples/resources/bloxone_ipam_range/resource.tf

```diff
@@ -0,0 +1,32 @@
+resource "bloxone_ip_space" "example_space" {
```

Comment:
should be `bloxone_ipam_ip_space`.. Similar comment for all resources in this example file. 

Also, the `_space` in `example_space` seems redundant. Same comment for the other names as well. It can be used if there is more than one resource of same type and we want to differentiate between them

Reply:
Made the suggested updates.